data/languages/kirk32.sinc

```diff
@@ -139,12 +139,12 @@ imm2: immed is t32_imm2 [immed = 0 + t32_imm2;] {
 	affectflags();
 }
 
-:rol addr1, imm2 is t32_opcode=0x90 & addr1 & imm2 {
-	addr1 = (addr1 << imm2) | (addr1 >> (32 - imm2));
+:shl addr1, imm2 is t32_opcode=0x90 & addr1 & imm2 {
```

Comment:
this breaks rom:000000fd

Reply:
hmm, I'm not sure why it does that, maybe it's more of an issue with mov16 because of word addressing

Reply:
fixed mov16, looks good now